l10n_it_vat_statement_communication/migrations/12.0.1.1.0/post-migrate.py

```diff
@@ -0,0 +1,25 @@
+#
+# Copyright (c) 2019, Link IT srl, Italy. All rights reserved.
+#
+
+import logging
+
+_logger = logging.getLogger(__name__)
+
+
+def migrate(cr, version):
+    if not version:
+        _logger.warning("""
+            Non è presente alcuna versione precedente del modulo.
+            Skippo la migration.
```

Comment:
Se compare nel .pot andrebbe in inglese.

P.S.: perdonami, ma "Skippo" non si può proprio vedere :smile: :kissing_heart: